libmscore/read301.cpp

```diff
@@ -262,6 +262,11 @@ bool Score::read(XmlReader& e)
       fixTicks();
       masterScore()->rebuildMidiMapping();
       masterScore()->updateChannel();
+
+      //TODO - PHV: is this a good place to do this?
```

Comment:
Not sure, but if it happens here, it probably needs to happen in read206 and read114 too.